src/test/controller/PromotionController.test.ts

```diff
@@ -259,13 +264,14 @@ describe('Unit tests for PromotionController', function () {
   test('POST /promotions/ - if restaurant with placeId does not exist in DB, promotion should create new restaurant', async (done) => {
     const user: User = new UserFactory().generate();
     const promotion = new PromotionFactory().generateWithRelatedEntities(user);
+    const inputtedPromotion = setGooglePlacesAddress(promotion);
```

Comment:
I think we can write some more tests
- one where restaurantAddress is empty string
- another where no restaurant exists and and we create restaurant with geocoder and check with the expected lat lon
- one where geocoder returns no results and for now, we create restaurant with null lat and null lon